client/gc_test.go

```diff
@@ -451,19 +485,23 @@ func TestAllocGarbageCollector_UsageBelowThreshold(t *testing.T) {
 
 func TestAllocGarbageCollector_UsedPercentThreshold(t *testing.T) {
 	t.Parallel()
-	logger := testLogger()
```

Comment:
Did you remove the testLogger func too? I may have missed that.

Reply:
Went through the entire code base and removed all calls to log.New in tests